.github/workflows/e2e-test.yml

```diff
@@ -573,16 +571,13 @@ jobs:
 
       - name: Install KServe
         run: |
+          export KSERVE_CUSTOM_ISVC_CONFIGS='ingress.pathTemplate=/serving/{{ .Namespace }}/{{ .Name }}|ingress.ingressDomain=mydomain.com'
```

Comment:
How are these configs being handled?

I am not sure, but patching the isvc sounds easier to debug and maintain. (if this conifg is related with the comment I left on another script that configures the isvc cm).